UploadServlet.java

```diff
@@ -160,5 +212,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			out.println(e);
 		}
 	}
+	
+	}  // else

```

Comment:
@hema-001 

What is  `// else`  (line 216) for?

Hui

Reply:
I have removed it, the change is outdated already as you can see above (the yellow rectangle).

Thanks.